lib/main_common.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:farmsmart_flutter/redux/app/app_state.dart';
 import 'package:farmsmart_flutter/redux/keys.dart';
 import 'package:farmsmart_flutter/redux/store.dart';
-import 'package:farmsmart_flutter/ui/home.dart';
+import 'package:farmsmart_flutter/ui/home_old.dart';
```

Comment:
We have a `home_old.dart` file in the project?

Reply:
### FILE REMOVED :D